packages/base/src/RTL.js

```diff
@@ -0,0 +1,32 @@
+import { getRTL, getLanguage } from "./Configuration";
+import getDesigntimePropertyAsArray from "./util/getDesigntimePropertyAsArray";
+import detectNavigatorLanguage from "./util/detectNavigatorLanguage";
+
+const M_ISO639_OLD_TO_NEW = {
+	"iw": "he",
+	"ji": "yi",
+	"in": "id",
+	"sh": "sr",
+};
+
+const A_RTL_LOCALES = getDesigntimePropertyAsArray("$cldr-rtl-locales:ar,fa,he$") || [];
+
+const impliesRTL = language => {
+	language = (language && M_ISO639_OLD_TO_NEW[language]) || language;
+
+	return A_RTL_LOCALES.indexOf(language) >= 0;
+};
+
+const getEffectiveRTL = () => {
+	const configurationRTL = getRTL();
+
+	if (configurationRTL !== null) {
+		return !!configurationRTL;
+	}
+
+	return impliesRTL(getLanguage() || detectNavigatorLanguage());
+};
+
+export default {};
```

Comment:
I don't think this is needed

Reply:
We have that enabled: https://github.com/benmosher/eslint-plugin-import/blob/v2.16.0/docs/rules/prefer-default-export.md

Should we have `#getEffectiveRTL` as default export?

Reply:
then let's rename the file to `getEffectiveRTL` if that is all it does, and use a default export. Unless we expect it to do more things later.